src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -294,8 +305,77 @@ protected Object decode(
                                 case 52:
                                     value = buf.readByte();
                                     position.set(
-                                            Position.PREFIX_TEMP + (j + 2 - 45),
-                                            (value != (byte) 0x80) ? value : null);
+                                            Position.PREFIX_TEMP + (j + 2 - 45), (value != (byte) 0x80) ? value : null);
+                                    break;
+                                case 53:
+                                    value = buf.readUnsignedShortLE();
+                                    if (value != 0x7FFF) {
+                                        if (BitUtil.check(value, 7)) {
+                                            position.set("fuelLevel", BitUtil.to(value, 6));
+                                        } else {
+                                            position.set(Position.KEY_FUEL_LEVEL, BitUtil.to(value, 6) / 10);
+                                        }
+                                    }
+                                    break;
+                                case 54:
+                                    position.set(Position.KEY_FUEL_USED, buf.readFloatLE());
+                                    break;
+                                case 55:
+                                    value = buf.readUnsignedShortLE();
+                                    position.set(Position.KEY_RPM, (value != 0xFFFF) ? value : null);
+                                    break;
+                                case 56:
+                                    value = buf.readByte();
+                                    position.set(Position.KEY_COOLANT_TEMP, (value != (byte) 0x80) ? value : null);
+                                    break;
+                                case 57:
+                                    position.set(Position.KEY_OBD_ODOMETER, buf.readFloatLE());
+                                    break;
+                                case 58:
+                                case 59:
+                                case 60:
+                                case 61:
+                                case 62:
+                                    value = buf.readUnsignedShortLE();
+                                    position.set(
+                                            Position.KEY_AXLE_WEIGHT + (j + 2 - 58), (value != 0xFFFF) ? value : null);
```

Comment:
This cannot be used as a prefix. Use string prefix instead.

Reply:
This string ok?

`obdAxleWeight`

Reply:
You can just use "axleWeight".

Reply:
Ok